### DISCUSSION TITLE
Option to start new translation in multiple components inside project at once

### DISCUSSION THREAD

**Discussion by maetiskou** [WeblateOrg/weblate#7195] [Q&A] [answered]
Hello,

I have a project with tens of components and i would like to add new language to selected few components in one go (maybe each one of them if possible). Is there any way to achieve this? Preferably in web UI?

**Comment by nijel** [comment, marked as answer]
There is an add-on to keep languages consistent: https://docs.weblate.org/en/latest/admin/addons.html#add-missing-languages

**Reply by maetiskou** [reply]
Thank you very much. This is quite applicable. Little side question. I cannot see _Add-ons_ tab under _Manage_ menu as shown in here: https://docs.weblate.org/en/latest/admin/addons.html. Could this be due to error in configuration or something?

**Reply by nijel** [reply]
You should see that if you have permissions. Note that these are at the component level, not at the project level.

**Reply by maetiskou** [reply]
Oh sure.. i was looking for it in project level _Manage_ menu.
